### DISCUSSION TITLE
Showcasing Selected Products from the CNCF Landscape

### DISCUSSION THREAD

**Discussion by Exnadella** [cncf/landscape2#462] [Ideas]
Please provide a template that allows companies to showcase products featured in the CNCF landscape on their websites. This template will facilitate investors and customers in quickly gaining an overview of recent advancements. It will also assist in exploring options for migrating to new tools or implementing new functionalities as soon as they become available.

1) App developers must have the capability to log in and choose a product from the displayed range of available products.
2) The CNCF should create a template for all chosen products, enabling the generation of PDFs, URLs, and images.
3) Current members should be able to monitor app developers and recommend tool integrations.

**Comment by tegioz** [comment]
Hi @Exnadella 👋

There is a mechanism to embed the items of a category or subcategory in the landscape on any website. You can find more details about how to set it up [here](https://github.com/cncf/landscape2#embeddable-views).

Please note that the CNCF landscape is a static site generated with the software in this repository (there are [more landscapes](https://github.com/cncf/landscape2-sites#landscape2-sites) that rely on it as well). This means that some features (i.e. log in, or some forms of dynamic content generation) may not fit well.
